Common/src/main/java/com/clinics/common/DTO/request/outer/doctor/AddEditAppointmentDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.clinics.common.DTO.request.outer.doctor;
+
+import lombok.*;
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@ToString
+public class AddEditAppointmentDTO {
```

Comment:
Bad naming. So it's Add or Edit ? Maybe it's enough to have AppointmentDTO

Reply:
Done